packages/itwin/grouping-mapping-widget/README.md

```diff
@@ -55,11 +60,48 @@ You can define your UI provider as follows in the GroupingMappingProvider:
 new GroupingMappingProvider({
   customUIs: [
     {
+      type: GroupingMappingCustomUIType.Grouping,
       name: "Manual",
       displayLabel: "Manual Query",
-      uiComponent: ManualGroupingUI,
+      uiComponent: ManualGroupingCustomUI,
       icon: <SvgDraw />,
     },
   ],
 });
 ```
+
+### Context Custom UI
+
+You can also add custom UI in context menu for each group. This could be UI driven interaction or simply a callback function to execute your own code.
+
+You can pass in `iModelId`, `mappingId` and `groupId` in `ContextCustomUIProps` to assist customized user experience.

```

Comment:
The user has to pass these IDs into the custom ui component, or does the widget pass them into the component that user created?

Reply:
the widget has passed in. Bad description, I will change it to 'You can use ...